packages/angular-cli/blueprints/ng2/files/__path__/test.ts

```diff
@@ -14,21 +19,15 @@ declare var require: any;
 // Prevent Karma from running prematurely.
 __karma__.loaded = function () {};
 
+// First, initialize the Angular testing environment.
```

Comment:
@hansl I tried throwing an error in a spec file (`throw new Error('error here');`) with and without the catch, and it's much better without it:
```
# with try/catch
kamik@T460p MINGW64 /D/sandbox/master-proj (master)
$ ng test --no-watch
09 11 2016 00:07:29.194:INFO [karma]: Karma v1.2.0 server started at http://localhost:9876/
09 11 2016 00:07:29.198:INFO [launcher]: Launching browser Chrome with unlimited concurrency
09 11 2016 00:07:29.280:INFO [launcher]: Starting browser Chrome
09 11 2016 00:07:30.590:INFO [Chrome 54.0.2840 (Windows 10 0.0.0)]: Connected on socket /#guzx9l6foj0Tw_5KAAAA with id 24284547
Chrome 54.0.2840 (Windows 10 0.0.0) ERROR


Chrome 54.0.2840 (Windows 10 0.0.0) ERROR



# without try/catch
kamik@T460p MINGW64 /D/sandbox/master-proj (master)
$ ng test --no-watch
09 11 2016 00:06:41.269:INFO [karma]: Karma v1.2.0 server started at http://localhost:9876/
09 11 2016 00:06:41.271:INFO [launcher]: Launching browser Chrome with unlimited concurrency
09 11 2016 00:06:41.279:INFO [launcher]: Starting browser Chrome
09 11 2016 00:06:42.600:INFO [Chrome 54.0.2840 (Windows 10 0.0.0)]: Connected on socket /#5bLXOqjWdAOdm5VyAAAA with id 53864020
Chrome 54.0.2840 (Windows 10 0.0.0) ERROR
  Uncaught Error: error here
  at webpack:///D:/sandbox/master-proj/src/app/app.component.spec.ts:35:0 <- src/test.ts:50522

Chrome 54.0.2840 (Windows 10 0.0.0) ERROR
  Uncaught Error: error here
  at webpack:///D:/sandbox/master-proj/src/app/app.component.spec.ts:35:0 <- src/test.ts:50522
```